nihms-filesystem-builder/src/main/java/org/dataconservancy/nihms/builder/fs/FilesystemModelBuilder.java

```diff
@@ -164,17 +158,9 @@ public NihmsSubmission build(String formDataUrl) throws InvalidModel {
             submission.setMetadata(metadata);
 
         } catch (IOException ioe) {
-            ioe.printStackTrace();
-        } finally {
-            if (is != null) {
-                try {
-                    is.close();
-                } catch (IOException ioe) {
-                    ioe.printStackTrace();
-                }
-            }
+            throw new InvalidModel(ioe.getMessage());
```

Comment:
In this case, the exception chain is being broken.  If an InvalidModel exception is thrown here, the developer will not know what the underlying exception was.  It's better to insure the exception is chained by performing:

`throw new InvalidModel(ioe.getMessage(), ioe);`